### ISSUE TITLE
Question: Is it fine to add URLs from another list (with evidences)?

### ISSUE THREAD

**Issue by S-trace** [quenhus/uBlock-Origin-dev-filter#38]
Hello.

Is it fine to add URLs from another list (with evidences)?

For example - will it be OK to submit a block request issue based on https://github.com/vladgba/Back2source/blob/master/back2source.user.js content (if I will provide evidence for each site in this list)?

I don't see any [license requirements](https://github.com/vladgba/Back2source/blob/master/LICENSE) from the origin of this list, but how about you?

Thank you.

**Comment by quenhus** [comment]
I would be really glad to add URLs from <https://github.com/vladgba/Back2source/>. We should ask @vladgba for approval, but he seems to agree (he liked your issue)

However, I really want to have evidence for every domains as I don't want to block useful websites, dead domains and I want to separate SO/GitHub/Wikipedia/ copycats.

From the list, it is not easy to determine what service is mirrored. We should search every domain with Google/DDG : "site:DOMAIN", to find evidence.

**Comment by quenhus** [comment]
I tried to clean the Back2source list by removing domain already in the current filter

Format: [domain] => [url extractedd from back2source]
```
360wiki.ru => *.360wiki.ru/wiki/*
abcdef.wiki => *.abcdef.wiki/*
answacode.com => *.answacode.com/questions/*
answeright.com => *.answeright.com/*
askdev.info => *.askdev.info/questions/*
askdev.ru => *.askdev.ru/q-*,*.askdev.ru/q/*
askentire.net => *.askentire.net/q/*-*
askvoprosy.com => *.askvoprosy.com/voprosy/*
bcqaw.com => *.bcqaw.com/*.html
bildiredi.com => *.bildiredi.com/*
bilee.com => *.bilee.com/*.html
buildwiki.ru => *.buildwiki.ru/wiki/*
ciupacabra.com => *.ciupacabra.com/*
codegear.dev => *.codegear.dev/*/questions/*
codeindex.ru => *.codeindex.ru/q/*
codengineering.ru => *.codengineering.ru/q/*
coderoad.in => *.coderoad.in/questions/*
coderoad.wiki => *.coderoad.wiki/*
coderquestion.ru => *.coderquestion.ru/q/*
datewiki.ru => *.datewiki.ru/wiki/*
de-vraag.com => *.de-vraag.com/*
devfaq.fr => *.devfaq.fr/question/*
dir.md => *.dir.md/*
donolik.com => *.donolik.com/*
dwf.life => *.dwf.life/*
e-learn.cn => *.e-learn.cn/topic/*
encyclopaedia.bid => *.encyclopaedia.bid/*
extutorial.com => *.extutorial.com/ask/*
fooobar.com => *.fooobar.com/questions/*
gaz.wiki => *.gaz.wiki/wiki/*
gitrush.ru => *.gitrush.ru/*/*/*
hmong.wiki => *.hmong.wiki/wiki/*
hmong.ru => *.hmong.ru/wiki/*
howtosolves.com => *.howtosolves.com/q/*
husl.ru => *.husl.ru/questions/*
icode9.com => *.icode9.com/*
issue.life => *.issue.life/questions/*
it-brain.online => *.it-brain.online/question/*
it-swarm.it => *.it-swarm.it/*/*
it-swarm-id.com => *.it-swarm-id.com/*/*
it-swarm-vi.com => *.it-swarm-vi.com/*/*
it-swarm.asia => *.it-swarm.asia/*/*
it-swarm.com.de => *.it-swarm.com.de/*/*
it-swarm.cn => *.it-swarm.cn/*/*
it-swarm.dev => *.it-swarm.dev/*/*
it-swarm-es.com => *.it-swarm-es.com/*/*
it-swarm.net => *.it-swarm.net/*/*
it-swarm.xyz => *.it-swarm.xyz/*/*
itnan.ru => *.itnan.ru/post.php*
itdaan.com => *.itdaan.com/*
itranslater.com => *.itranslater.com/qa/details/*
jejakjabar.com => *.jejakjabar.com/wiki/*
legkovopros.ru => *.legkovopros.ru/questions/*
livepcwiki.ru => *.livepcwiki.ru/wiki/*
mihalicdictionary.org => *.mihalicdictionary.org/*
mlink.in => *.mlink.in/*
mlog.club => *.mlog.club/article/*
nina.az => *.*.nina.az/wiki/*
overcoder.ru => *.overcoder.ru/q/*
prog-help.ru => *.prog-help.ru/*
qa-help.ru => *.qa-help.ru/*
qacode.ru => *.qacode.ru/questions/*
qarchive.ru => *.qarchive.ru/*
qaru.tech => *.qaru.tech/questions/*
qarus.ru => *.qarus.ru/*
qastack.lk => *.qastack.lk/*
qastack.pl => *.qastack.pl/*
recalll.co => *.recalll.co/*
rudata.ru => *.rudata.ru/wiki/*
sbup.com => *.sbup.com/wiki/*
savepearlharbor.com => *.savepearlharbor.com/?p=*
sobrelinux.info => *.sobrelinux.info/questions/*
soinside.com => *.soinside.com/question/*
stackanswers.net => *.stackanswers.net/questions/*
stackoverflood.com => *.stackoverflood.com/*
switch-case.com => *.switch-case.com/*
techarks.ru => *.techarks.ru/qa/*
techfeed.net => *.techfeed.net/*
territorioscuola.it => *.territorioscuola.it/*
thinbug.com => *.thinbug.com/q/*
utyatnishna.ru => *.utyatnishna.ru/info/*
v-resheno.ru => *.v-resheno.ru/*
vvikipedla.com => *.vvikipedla.com/wiki/*
web-answers.ru => *.web-answers.ru/*/*
while-do.com => *.while-do.com/*
wiki-org.ru => *.wiki-org.ru/*
wiki-wiki.ru => *.wiki-wiki.ru/wp/*
wiki2.net => *.wiki2.net/*
wiki2.online => *.wiki2.online/*
wiki2.info => *.wiki2.info/*
wiki2.org => *.wiki2.org/*
wiki.cologne => *.wiki.cologne/*
wiki2.wiki => *.wiki2.wiki/wiki/*
wikichi.ru => *.wikichi.ru/wiki/*
wikies.wiki => *.wikies.wiki/wiki/*
wikipe.wiki => *.wikipe.wiki/wiki/*
wikipedia-on-ipfs.org => *.wikipedia-on-ipfs.org/wiki/*
wikipedia.tel => *.wikipedia.tel/*
wikidark.ru => *.wikidark.ru/*
wikiredia.ru => *.wikiredia.ru/*
wikipedia24.ru => *.wikipedia24.ru/*
wikivisually.com => *.wikivisually.com/wiki/*
wikiwand.com => *.wikiwand.com/*/*
xbuba.com => *.xbuba.com/*
xcv.wiki => *.xcv.wiki/*
xszz.org => *.xszz.org/*/question-*
ylhow.com => *.ylhow.com/*
yuanmacha.com => *.yuanmacha.com/*.html
zapytay.com => *.zapytay.com/*
qa.1r1g.com => qa.1r1g.com/sf/ask/*
ru.encyclopedia.kz => ru.encyclopedia.kz/index.php/*
askfrance.me => askfrance.me/*
respuestas.me => respuestas.me/*
antwortenhier.me => antwortenhier.me/*
pythonq.com => pythonq.com/*/*/*
proubuntu.ru => proubuntu.ru/*/*
cloud.tencent.com => cloud.tencent.com/developer/ask/*"
```

For each domain, we should find evidence and determine if it mirrors SO/GH/Wikipedia/...

**Comment by vladgba** [comment]
Hello. 
I agree to add sites. 
I'll try tomorrow to compile full lists with SO/Wiki clones. 
Back2source also has redirects from aggregator sites and clones of other sites.  Do these sites need to be on the lists?

**Comment by quenhus** [comment]
Thanks a lot @vladgba! I didn't want to add to your workload, I was only asking for the permission to use your blocklist. Of course, your help is highly appreciated. 

Currently, this filter only support StackOverflow/GitHub/NPM/Wikipedia mirrors. For now, I would like to concentrate only on copycats bothering developers, usually those using SEO techniques to rank first on search engines.

**Comment by vladgba** [comment]
Some of the sites were not working. 
Here are the resulting lists:
[so.txt](https://github.com/quenhus/uBlock-Origin-dev-filter/files/8109007/so.txt)
[wiki.txt](https://github.com/quenhus/uBlock-Origin-dev-filter/files/8109008/wiki.txt
)
[other-stackexchange.txt](https://github.com/quenhus/uBlock-Origin-dev-filter/files/8109012/other-stackexchange.txt)

**Comment by quenhus** [comment]
Thanks for the amazing work on that.

If it is OK for you, I'm going to take the lists you just give us and integrate them in this repo. (I just want to ensure every copycats added respect [these conditions](https://github.com/quenhus/uBlock-Origin-dev-filter#scope-of-this-filter))

**Comment by quenhus** [comment]
Thanks again for everything.

Closed by 3cdfb08e7d07323c5c7186caf7d99d20307ffa09 (Wikipedia) and 189e808ca342914f688231ae5939b21e5b55fd2b (SO)
